src/vm/exceptionhandling.h

```diff
@@ -389,15 +389,15 @@ class ExceptionTracker
 
     bool IsStackOverflowException();
 
-#ifdef FEATURE_PAL
+#if defined(FEATURE_PAL) && !defined(CROSS_COMPILE)
```

Comment:
This method is part of the exception handling that's used by all Linux targets. So I don't understand why is it being removed.